frontend/src/libs/components/app/app.tsx

```diff
@@ -15,6 +16,12 @@ const App: React.FC = () => {
     }
   }, [isRoot, dispatch]);
 
+  useEffect(() => {
+    if (isRoot) {
```

Comment:
Guess it shouldn't have a check if we are on root route, but have a check if there is no user in store

Reply:
right. add check in the action